plugins/woocommerce/tests/e2e-pw/tests/shopper/mini-cart.spec.js

```diff
@@ -6,6 +6,7 @@ const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
 const pageTitle = 'Mini Cart';
 const pageSlug = pageTitle.replace( / /gi, '-' ).toLowerCase();
 const miniCartButton = '.wc-block-mini-cart__button';
+const miniCartButton = '.wc-block-mini-cart__badge';
```

Comment:
I think there's typo

```suggestion
const miniCartBadge = '.wc-block-mini-cart__badge';
```

Reply:
Oh, right. I just fixed it!